ecommerce/CartController.php

```diff
@@ -76,28 +76,43 @@ public function __construct( $register_hooks = true ) {
 
 		if ( $register_hooks ) {
 			/**
-			 * Handle AJAX request for getting cart related data by cart ID.
+			 * Handle AJAX request for adding course to cart
 			 *
 			 * @since 3.0.0
 			 */
-			add_action( 'wp_ajax_tutor_cart_details', array( $this, 'get_cart_by_id' ) );
+			add_action( 'wp_ajax_tutor_add_course_to_cart', array( $this, 'tutor_add_course_to_cart' ) );
```

Comment:
We don't need to prefix **tutor** in internal method name `tutor_add_course_to_cart` since we already have namespace